agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/configuration/Configuration.java

```diff
@@ -216,6 +216,7 @@ public static class JdbcMasking {
 
   public static class LoggingInstrumentation {
     public String level = "INFO";
+    public boolean captureCodeAttributes;
```

Comment:
it may be good to put it under logback-specific preview configuration, until we support across all logging instrumentations, e.g.

```
{
  "preview": {
    "captureLogbackCodeAttributes": true
  }
}
```